list_books.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
+
+$query = "SELECT * FROM books";
```

Comment:
The list_books.php file also directly executes SQL queries without using prepared statements. Although this is less risky since it doesn't involve user input, it's a good practice to consistently use prepared statements for all database interactions.

Consider refactoring the query execution into a repository pattern as shown in previous examples.